packages/docs/components/sidebar/Sidebar.vue

```diff
@@ -1,64 +1,162 @@
 <template>
-  <va-sidebar :minimized="minimized">
-    <template slot="menu">
-      <template v-for="(item, key) in items">
-        <va-sidebar-link-group
-          v-if="item.children"
+  <va-sidebar class="sidebar" color="secondary" v-model="minimized">
```

Comment:
Navigation is not reactive:
![image](https://user-images.githubusercontent.com/5394573/95446494-0e3c9700-0969-11eb-8b35-e62d3562642f.png)